# VAERS_Data_Scrubber
Scrubbed VAERS Covid-19 data into pandas dateframe.

NOTE: if pd.read_csv(filePath) throws an error, open the source .csv files, and save them. Doing this alleviates the bug, not sure why. Probably needs a specific encoding param.

https://vaers.hhs.gov/data.html

To run the script
- Download/ fork repo
- Download the files in the link
- Check the package requirements at beginning of script
- Update the file paths
- Run the whole thing


P.S If you ever read this, cheers Logan you saved me a f&@k load of time. 
